<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_crear_platillo</class>
 <widget class="QDialog" name="admin_crear_platillo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>600</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QStackedWidget" name="sW_datosPlatillo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>651</width>
     <height>541</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="elPatio_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>114</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(133, 133, 133);
</string>
     </property>
     <property name="text">
      <string>Datos generales</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_fotografia">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>128</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(133, 133, 133);
border: 1px solid rgb(133, 133, 133);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Añada una foto</string>
     </property>
    </widget>
    <widget class="QLabel" name="elPatio_5">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>99</y>
       <width>74</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(133, 133, 133);
</string>
     </property>
     <property name="text">
      <string>Fotografía</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>259</x>
       <y>340</y>
       <width>351</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="elPatio_9">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>Precio (MNX)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_precio">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
border-bottom: 2px solid #539E56;
border-radius: 3px;</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_editarFoto">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>340</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	color: #539E56;
	border-radius: 10px;
	background-color: #E3EFB0;
	border: none;
	padding: 5px;
}

/*
:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}
*/</string>
     </property>
     <property name="text">
      <string>Editar</string>
     </property>
     <property name="icon">
      <iconset resource="../Recursos.qrc">
       <normaloff>:/iconos/resources/editarPlato.png</normaloff>:/iconos/resources/editarPlato.png</iconset>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>361</width>
       <height>65</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="elPatio_6">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_nombrePlatillo">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
border-bottom: 2px solid #539E56;
border-radius: 3px;</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>361</width>
       <height>67</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="elPatio_7">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>Categoria</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="cb_categoria">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
border-bottom: 2px solid #539E56;
border-radius: 3px;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_nuevaCategoria">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">*{
	color: #539E56;
	border-radius: 10px;
	background-color: #E3EFB0;
	border: none;
	padding: 5px;
}

/*
:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}
*/</string>
           </property>
           <property name="text">
            <string>Nueva categoria</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_categoria">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
border-bottom: 2px solid #539E56;
border-radius: 3px;</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_canNueCategoria">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">*{
	color: #539E56;
	border-radius: 10px;
	background-color: #E3EFB0;
	border: none;
	padding: 5px;
}

/*
:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}
*/</string>
           </property>
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>259</x>
       <y>240</y>
       <width>351</width>
       <height>92</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="elPatio_8">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>Descripción</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="te_descripcion">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
border-bottom: 2px solid #539E56;
border-radius: 3px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_siguiente">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	color: white;
	border-radius: 10px;
	background-color: #46B04A;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}

:disabled {
	background-color: rgb(133, 133, 133);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Sigueinte</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="elPatio_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>126</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(133, 133, 133);
</string>
     </property>
     <property name="text">
      <string>Datos especificos</string>
     </property>
    </widget>
    <widget class="QLabel" name="elPatio_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>86</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(133, 133, 133);
</string>
     </property>
     <property name="text">
      <string>Ingredientes</string>
     </property>
    </widget>
    <widget class="QLabel" name="elPatio_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>181</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(133, 133, 133);
</string>
     </property>
     <property name="text">
      <string>Lista de Ingredientes existentes</string>
     </property>
    </widget>
    <widget class="QLabel" name="elPatio_14">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>181</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(133, 133, 133);
</string>
     </property>
     <property name="text">
      <string>Ingredientes del platillo</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>480</y>
       <width>208</width>
       <height>37</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnCancelar">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
	color: #539E56;
	border-radius: 10px;
	background-color: #E3EFB0;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCrearPlatillo">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
	color: white;
	border-radius: 10px;
	background-color: #46B04A;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}

:disabled {
	background-color: rgb(133, 133, 133);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Crear Platillo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_atras">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	color: white;
	border-radius: 10px;
	background-color: #46B04A;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}

:disabled {
	background-color: rgb(133, 133, 133);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Atras</string>
     </property>
    </widget>
    <widget class="QLabel" name="elPatio_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>430</y>
       <width>241</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(133, 133, 133);
</string>
     </property>
     <property name="text">
      <string>*da doble click para agregar un ingrediente</string>
     </property>
    </widget>
    <widget class="QLabel" name="elPatio_16">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>430</y>
       <width>241</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(133, 133, 133);
</string>
     </property>
     <property name="text">
      <string>*da doble click para quitar un ingrediente</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_ingreExistente">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>281</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="list_ingrePlatillo">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>281</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="elPatio_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>21</y>
     <width>204</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #585858
</string>
   </property>
   <property name="text">
    <string>Agregar nuevo Platillo</string>
   </property>
  </widget>
  <widget class="QLabel" name="elPatio_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>52</y>
     <width>283</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(133, 133, 133);
</string>
   </property>
   <property name="text">
    <string>Ingresa la información del nuevo platillo</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
